src/machine/usb/cdc/usbcdc.go

```diff
@@ -0,0 +1,215 @@
+package cdc
+
+import (
+	"errors"
+	"machine"
+	"runtime/interrupt"
+	"runtime/volatile"
+)
+
+var (
+	errUSBCDCBufferEmpty      = errors.New("USB-CDC buffer empty")
```

Comment:
These should probably be capitalized so they can be exported. Or perhaps all the errors should be in the `usb` package itself so they can be more easily referred to from user programs?

Reply:
The error definitions can stay in the sub-packages as long as they are exported that is still a logical point of utilization for user code.

Reply:
I have corrected the error name.